src/PhpPact/Standalone/MockService/MockServer.php

```diff
@@ -122,7 +118,7 @@ private function verifyHealthCheck(): bool
             try {
                 return $service->healthCheck();
             } catch (ConnectionException $e) {
-                \sleep($retrySec);
+                \usleep(round($retrySec * 1000000));
```

Comment:
round returns a float and usleep expects an int

https://www.php.net/manual/en/function.round.php
https://www.php.net/manual/en/function.usleep.php

`usleep(intval(round($retrySec * 1000000)));`